utils/build/docker/java/spring-boot/src/main/java/com/datadoghq/system_tests/springboot/debugger/DebuggerController.java

```diff
@@ -144,4 +144,13 @@ public String exceptionReplayRecursion20(@RequestParam(required = false, default
             throw new ResponseStatusException(org.springframework.http.HttpStatus.INTERNAL_SERVER_ERROR, "Recursion exception");
         }
     }
+
+    @GetMapping("/exceptionreplay/inner")
+    public Void exceptionReplayInner() {
+        try {
+            throw new ResponseStatusException(org.springframework.http.HttpStatus.INTERNAL_SERVER_ERROR, "Inner exception");
+        } catch (RuntimeException ex) {
+            throw new ResponseStatusException(org.springframework.http.HttpStatus.INTERNAL_SERVER_ERROR, "Outer exception");
+        }
+    }
```

Comment:
I don't think the inner exception is wrapped, there is no reference to `ex`

Reply:
Oh right, my bad. Fixed here - a2d065f3d14dfe7e128299a7efa5b9a332f50e9b